### PR TITLE
[#173311623] Move counterfeit app files to Coinhouse

### PR DESCRIPTION
**Pull request by BenjaminG**
https://www.pivotaltracker.com/story/show/173311623

### CODE REVIEW

**Review by unflores — APPROVED** [review]
Looks good. Remember you'll need to update your gemfile in the monolith for this to take action.

File: README.md [hunk 1]
```diff
@@ -1,8 +1,6 @@
 # Counterfeit
 
-Welcome to your new gem! In this directory, you'll find the files you need to be able to package up your Ruby library into a gem. Put your Ruby code in the file `lib/counterfeit`. To experiment with that code, run `bin/console` for an interactive prompt.
-
-TODO: Delete this and the text above, and describe your gem
+Simple tool based on [Webmock](https://github.com/bblimke/webmock) and [Sinatra](https://github.com/sinatra/sinatra) to quickly mock API endpoints
```
Comment (unflores) [inline README.md:3]: stub endpoints not mock them.

File: lib/counterfeit.rb [hunk 2]
```diff
@@ -1,35 +1,13 @@
 require 'webmock'
 require 'active_support'
 require 'active_support/core_ext'
-require 'counterfeit/version'
```
Comment (vrenaudineau) [inline lib/counterfeit.rb:4]: Shouldn't we keep version ?

File: lib/counterfeit/info/app.rb [hunk 3]
```diff
@@ -1,16 +0,0 @@
-require 'sinatra/base'
-require 'haml'
-
-module Counterfeit
-  module Info
```
Comment (vrenaudineau) [inline lib/counterfeit/info/app.rb:5]: Isn't this supposed to stay, like a status service ?

File: lib/counterfeit/info/views/index.haml [hunk 4]
```diff
@@ -1,16 +0,0 @@
-!!!  XML
```
Comment (vrenaudineau) [inline lib/counterfeit/info/views/index.haml:1]: This file is needed to render `haml :index`

**Review by vrenaudineau — CHANGES_REQUESTED** [review]

**Review by vrenaudineau — APPROVED** [review]

**Pull request merged**